src/ztex/device_format.c

```diff
@@ -79,15 +76,35 @@ void device_format_init(struct fmt_main *fmt_main,
 	jtr_devices_allow = devices_allow;
 	jtr_verbosity = verbosity;
 
-	struct list_entry *entry;
-	int found_bad_sn = 0;
-	for (entry = devices_allow->head; entry; entry = entry->next)
-		if (!ztex_sn_is_valid(entry->data)) {
-			fprintf(stderr, "Error: bad Serial Number '%s'\n", entry->data);
-			found_bad_sn = 1;
+
+	static int init_conf_devices = 0;
+	if (!init_conf_devices) {
+

```

Comment:
Inconsistent empty line.

Reply:
I use to insert a blank line where I feel code is too condensed, hard to read/understand, however I'm not ready to provide a formalized criteria for "density". Could you please explain what is "inconsistency" here from your point of view?

Reply:
I think/hope we never have empty lines between the `if` or such and the following code block, like you did here and maybe in your other code contributions. To me, that empty line breaks the logical flow of code and makes the code harder to read, not easier.

We sometimes have blank lines _before_ the `if` in cases where that block needs to be separated for easier reading. In your case, the blank line before `static int init_conf_devices = 0;` looks sufficient to separate this portion of code from what preceded it.

Reply:
On another note we currently require declarations to be in top of block in the rest of Jumbo, so this `static int init_conf_devices` will emit a `-Wdeclaration-after-statement` warning (or even error with `-Werror`). I have no plans to enforce that for the ZTEX code though, perhaps we should even drop that requirement some day.